src/subdomains/core/sell-crypto/sell/sell.controller.ts

```diff
@@ -95,20 +100,20 @@ export class SellController {
 
   private async toPaymentInfoDto(userId: number, sell: Sell, dto: GetSellPaymentInfoDto): Promise<SellPaymentInfoDto> {
     return {
-      fee: await this.getFee(userId, dto.asset),
+      ...(await this.getFee(userId, dto.asset)),
       depositAddress: sell.deposit.address,
       blockchain: sell.deposit.blockchain,
       minDeposits: this.getMinDeposit(sell.fiat.name),
     };
   }
 
   // --- HELPER-METHODS --- //
-  async getFee(userId: number, asset?: Asset): Promise<number> {
+  async getFee(userId: number, asset?: Asset): Promise<{ fee: number }> {
     if (asset) {
       asset = await this.assetService.getAssetById(asset.id);
-      if (asset?.category === AssetCategory.STOCK) return 0;
+      if (asset?.category === AssetCategory.STOCK) return { fee: 0 };
```

Comment:
Move this also into `userService. getUserSellFee`, to have everything in one place?